introduction_to_amazon_algorithms/jumpstart_named_entity_recognition/Amazon_JumpStart_Named_Entity_Recognition.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<p>Can we name notebooks in lowercase snakecase for both notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Is there a convention/rule here?
Exact naming was decided in consultation with our GM. So, if it is not a very hard rule, we should go ahead with the current convention.